Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -2050,65 +2049,72 @@ def run_script_execute_commands_command(client: CoreClient, args: Dict) -> Comma
     )
 
 
-def run_script_kill_process_command(client: CoreClient, args: Dict) -> List[CommandResults]:
+def run_script_kill_process_command(client: CoreClient, args: Dict) -> CommandResults:
     endpoint_ids = argToList(args.get('endpoint_ids'))
     incident_id = arg_to_number(args.get('incident_id'))
     timeout = arg_to_number(args.get('timeout', 600)) or 600
     processes_names = argToList(args.get('process_name'))
-    all_processes_response = []
+    replies = []
+
     for process_name in processes_names:
         parameters = {'process_name': process_name}
         response = client.run_script('fd0a544a99a9421222b4f57a11839481', endpoint_ids, parameters, timeout, incident_id)
         reply = response.get('reply')
-        all_processes_response.append(CommandResults(
-            readable_output=tableToMarkdown(f'Run Script Kill Process on {process_name}', reply),
-            outputs_prefix=f'{args.get("integration_context_brand", "CoreApiModule")}.ScriptRun',
-            outputs_key_field='action_id',
-            outputs=reply,
-            raw_response=reply,
-        ))
+        replies.append(reply)
+
+    command_result = CommandResults(
+        readable_output=tableToMarkdown(f'Run Script Kill Process on {",".join(processes_names)}', replies),
```

Comment:
does a reply contain the process name? if so, this is good enough:
```suggestion
        readable_output=tableToMarkdown("Run Script Kill Process Results", replies),
```
similar below